fastapi_backend/tests/conftest.py

```diff
@@ -58,7 +58,9 @@ async def override_get_async_session():
     app.dependency_overrides[get_user_db] = override_get_db
     app.dependency_overrides[get_async_session] = override_get_async_session
 
-    async with AsyncClient(app=app, base_url="http://localhost:8000") as client:
+    async with AsyncClient(
+        transport=ASGITransport(app=app), base_url="http://localhost:8000"
+    ) as client:
```

Comment:
Why this was changed? Please add comments on your own PRs to explain the reviewer about any non-trivial unrelated changes.

Reply:
It was deprecated, I got the warning and updated it.